charters/QueryService/charter.md

```diff
@@ -0,0 +1,79 @@
+# Query Service
+
+## Description
+The Query Service is an Application Programming Interface (API) for querying Human Cell Atlas (HCA) bundle metadata with intuitive query syntax.
+
+Services the Query Module provides may complement or subsume the current search functionality in the Data Storage Service (DSS).
+
+## Objectives
+1. Build, operate, and maintain a metadata API query service that allows a variety of queries to be performed against bundle metadata stored in HCA Data Coordination Platform (DCP) DSS
+1. Develop a query language specification for intuitive querying of HCA metadata
```

Comment:
Define the target users of intuitive would bound this objective.  I would assume it is programmers, not end-user.